### DISCUSSION TITLE
add geologicalContext terms to public data exports

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#219] [Wish List]
### Symbiota portal name

Pteridoportal

### What happened?

Fields in the Darwin Core class geologicalContext are not currently exporting when data are downloaded from the public search interface: [native-vs-dwc-output-comparison_2022-10-12.xlsx](https://github.com/BioKIC/symbiota-docs/files/9768030/native-vs-dwc-output-comparison_2022-10-12.xlsx)

This includes both Darwin Core Archives and Symbiota native exports.

### What operational systems are you seeing the problem on?

MacOS

### What browsers are you seeing the problem on?

Chrome

**Comment by themerekat** [comment]
This was probably just never developed
